### PR TITLE
Lint markdown files syntax for v5.0 with the new linter

### PR DESCRIPTION
**Pull request by iAdramelk**
Upd: Added Microsoft link to whitelist and remove releases table

### CODE REVIEW

**Comment by klizhentas** [comment]
@iAdramelk please remove releases-table and it's contents from the docs, it points to obsolete downloads anyways

File: docs/pages/preview/database-access.mdx [hunk 1]
```diff
@@ -357,10 +380,10 @@ Teleport Database Access is available starting from `6.0.0-alpha.1` pre-release.
 Download the appropriate version of Teleport for your platform from the table
 below, or visit our [downloads page](https://goteleport.com/teleport/download).
 
-{!docs/pages/preview/releases-table.mdx!}
```
Comment (klizhentas) [inline docs/pages/preview/database-access.mdx:360]: remove the page and the include for tests to pass

**Review by klizhentas — CHANGES_REQUESTED** [review]
Remove the releases-table.mdx for the tests to pass.

Escape a schema link (not a real link) for the tests to pass:

`http://schemas.microsoft.com/ws/2008/06/identity/claims/windowsaccountname`

File: docs/pages/preview/database-access.mdx (hunk 1 above)
Comment (iAdramelk) [reply docs/pages/preview/database-access.mdx:360]: Should we remove all includes in all versions? In 6.0/6.1  `release-table.mdx` is imported 5 times:

- https://github.com/gravitational/teleport/blob/branch/v6/docs/pages/database-access/getting-started.mdx
- https://github.com/gravitational/teleport/blob/branch/v6/docs/pages/database-access/guides/mysql-aws.mdx
- https://github.com/gravitational/teleport/blob/branch/v6/docs/pages/database-access/guides/mysql-self-hosted.mdx
- https://github.com/gravitational/teleport/blob/branch/v6/docs/pages/database-access/guides/postgres-aws.mdx
- https://github.com/gravitational/teleport/blob/branch/v6/docs/pages/database-access/guides/postgres-self-hosted.mdx

In other branches it's also imported multiple times.

**Comment by iAdramelk** [comment]
Added Microsoft link to whitelist and remove releases table. Please check https://github.com/gravitational/teleport/pull/5876/commits/a3d771f94357a9bd5b84c547b395d64b9108c8f6

**Review by klizhentas — APPROVED** [review]

**Comment by klizhentas** [comment]
@benarent @webvictim please approve

**Review by benarent — APPROVED** [review]

**Pull request merged**